src/presentational/iaux-button.ts

```diff
@@ -0,0 +1,96 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+
+import '@internetarchive/icon-donate/icon-donate.js';
+
+@customElement('iaux-button')
+export class IauxButton extends LitElement {
+  @property({ type: Boolean, reflect: true }) isDisabled = false;
+
+  @property({ type: Function }) clickHandler: Function | undefined;
+
+  @query('button') button!: HTMLButtonElement;
+
+  render() {
+    return html`
+      <button
+        ?disabled=${this.isDisabled}
+        @click=${() => {
+          if (this.isDisabled) {
```

Comment:
Do you need this disabled check? The `?disabled` above should be making it unclickable, no?